setup.py

```diff
@@ -110,4 +109,6 @@ def read(*rnames):
     ],
     include_package_data=True,
     zip_safe=False,
+    # XXX: This doesn't pick up the doctests.
+    test_suite='zope.file.tests',
```

Comment:
> OTOH python setup.py test is a disappointment, so I'm not sure it's worth the trouble to support it correctly.

Agreed. I run into namespace package issues still with that sometimes. I'll take it out.